.github/workflows/create-deployment-spack.yml

```diff
@@ -67,4 +67,10 @@ jobs:
           git -C ${{ env.ROOT_VERSION_LOCATION }} clone https://github.com/ACCESS-NRI/spack-config.git --branch ${{ inputs.spack-config-version }}
           ln -s -r -v ${{ env.ROOT_VERSION_LOCATION }}/spack-config/v${{ steps.strip.outputs.version-dir }}/${{ vars.DEPLOYMENT_TARGET }}/* ${{ env.ROOT_VERSION_LOCATION }}/spack/etc/spack/
           mkdir ${{ env.ROOT_VERSION_LOCATION }}/release
+
+          # Create restricted folders
+          mkdir -p ${{ env.ROOT_VERSION_LOCATION }}/restricted/ukmo/release
+          mkdir -p ${{ env.ROOT_VERSION_LOCATION }}/restricted/ukmo/source_cache
```

Comment:
Whaddabout the build stage in `/scratch`?

Reply:
Added